bps/clustering/clustering_ApPipeWithFakes.yaml

```diff
@@ -19,8 +19,7 @@
 clusterAlgorithm: lsst.ctrl.bps.quantum_clustering_funcs.dimension_clustering
 cluster:
   singleFrame:
-    # TODO DM-40388: replace this with isr,calibrateImage
-    pipetasks: isr,characterizeImage,calibrate
+    pipetasks: isr,calibrateImage
```

Comment:
These have to stay as-is: the Fakes pipeline isn't ready to use CalibrateImage. I just filed [DM-43351](https://rubinobs.atlassian.net/browse/DM-43351) for this. Please just change the TODO to reference that ticket.